virt/virtual_machines/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-configuring-USB-host-passthrough"]
+= Configuring USB host passthrough
+include::_attributes/common-attributes.adoc[]
+:context: virt-configuring-USB-host-passthrough
```

Comment:
Also not something to stop merge review over, but it would be worth fixing in a later PR - USB in both the id and context should probably be lowercase per our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata) about assembly file metadata:
```suggestion
[id="virt-configuring-usb-host-passthrough"]
= Configuring USB host passthrough
include::_attributes/common-attributes.adoc[]
:context: virt-configuring-usb-host-passthrough
```